src/ontology/mondo-edit.obo

```diff
@@ -361087,32 +361054,52 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/537/acute-no
 synonym: "Li-Fraumeni familial cancer susceptibility syndrome" EXACT [NCIT:C3476]
 synonym: "Li-Fraumeni familiar cancer susceptibility syndrome" EXACT [DOID:3012, NCIT:C3476]
 synonym: "Li-Fraumeni syndrome" EXACT CLINGEN_PREFERRED [NCIT:C3476]
+synonym: "Li-Fraumeni syndrome" RELATED [OMIM:151623]
+synonym: "Li-Fraumeni syndrome caused by mutation in TP53" EXACT [MONDO:design_pattern]
+synonym: "Li-Fraumeni-like syndrome" RELATED [OMIM:151623]
+synonym: "sarcoma family syndrome of 51 and Fraumeni" RELATED [OMIM:151623]
 synonym: "sarcoma family syndrome of Li and Fraumeni" RELATED [GARD:0006902]
 synonym: "sarcoma, breast, leukaemia and adrenal gland syndrome" EXACT [DOID:3012]
 synonym: "SBLA syndrome" EXACT [DOID:3012]
+synonym: "Sbla syndrome" RELATED [OMIM:151623]
 synonym: "SBLA syndrome (sarcoma, breast, leukemia, and adrenal gland)" RELATED [GARD:0006902]
+synonym: "TP53 Li-Fraumeni syndrome" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
+xref: DOID:0111503
 xref: DOID:3012 {source="MONDO:equivalentTo"}
 xref: GARD:0006902 {source="MONDO:equivalentTo"}
 xref: ICD10CM:D48.9 {source="Orphanet:524", source="Orphanet:524/attributed", source="Orphanet:524/ntbt"}
 xref: ICD9:V84.01 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MedDRA:10066795 {source="Orphanet:524", source="Orphanet:524/e"}
+xref: MESH:C563754
 xref: MESH:D016864 {source="DOID:3012", source="Orphanet:524", source="MONDO:equivalentTo", source="Orphanet:524/e"}
 xref: NCIT:C3476 {source="DOID:3012", source="MONDO:equivalentTo", source="MONDO:exact-label-match"}
-xref: OMIMPS:151623 {source="MONDO:cjm", source="MONDO:equivalentTo"}
+xref: OMIM:151623 {source="MONDO:equivalentTo"}
+xref: OMIM:609266 {source="MONDO:equivalentObsolete"}
+xref: OMIMPS:151623 {source="MONDO:cjm"}
+xref: Orphanet:524 {source="OMIM:151623", source="OMIM:609266"}
 xref: Orphanet:524 {source="DOID:3012", source="MONDO:equivalentTo"}
 xref: SCTID:428850001 {source="DOID:3012", source="MONDO:equivalentTo"}
 xref: UMLS:C0085390 {source="DOID:3012", source="Orphanet:524", source="MONDO:equivalentTo", source="Orphanet:524/e", source="NCIT:C3476"}
+xref: UMLS:C1835398 {source="OMIM:151623", source="MONDO:ncbi_mim2gene_medline"}
+xref: UMLS:C1836481 {source="MONDO:ncbi_mim2gene_medline", source="OMIM:609266"}
+xref: UMLS:C2675080 {source="OMIM:151623", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0000426 {source="DOID:3012", source="MONDO:Redundant"} ! autosomal dominant disease
 is_a: MONDO:0015356 {source="MESH:D016864", source="NCIT:C3476", source="Orphanet:524"} ! hereditary neoplastic syndrome
 is_a: MONDO:0016756 {source="Orphanet:524"} ! inherited nervous system cancer-predisposing syndrome
-relationship: has_characteristic HP:0000006 {source="Orphanet:524"} ! Autosomal dominant inheritance
+relationship: has_characteristic HP:0000006 {source="MONDO:HPOA", source="OMIM:151623", source="Orphanet:524"} ! Autosomal dominant inheritance
```

Comment:
I've never seen source MONDO:HPOA before

Reply:
I have never seen it either. I am surprised it is not throwing an error.

Reply:
yeah, it's weird. I'll leave it though.